app/controllers/content_types/school_home_pages_controller.rb

```diff
@@ -33,7 +33,7 @@ def slideshow
         'COMMON STYLES' => cascade_block('_cascade/blocks/html/common_styles'),
         'COMMON SCRIPTS' => cascade_block('_cascade/blocks/html/common_scripts'),
         'CUSTOM THEME' => cascade_block('_cascade/blocks/custom_themes/law'),
-        'GLOBAL FOOTER' => cascade_block('_cascade/blocks/html/law_footer'),
+        'GLOBAL FOOTER' => cascade_block('widgets/shared/law_footer'),
```

Comment:
Well in Cascade, both the global footer and the law footer are under `_cascade/blocks/html/`, but then in cascade-assets the global footer is under `widgets/shared/` and the law footer is under `_cascade/blocks/html/`